os-podman.te

```diff
@@ -13,6 +13,7 @@ gen_require(`
         type swift_var_cache_t;
         type fixed_disk_device_t;
         class blk_file getattr;
+        class socket_file;
```

Comment:
This doesn't seem to be used anywhere, since we're using the macro? I don't think it's necessary.

Reply:
hmmmm not sure - isn't it needed by the manage_sock_files_pattern() ? or does it add that class directly? I can of course remove this line, but..

Reply:
I tested the patch on 8 and 9 without it, for what it's worth. My understanding is that the definitions are only needed for the rules explicitly written out in the policy file.